nilearn/_utils/niimg_conversions.py

```diff
@@ -33,7 +32,56 @@ def _check_same_fov(img1, img2):
             and np.allclose(img1.get_affine(), img2.get_affine()))
 
 
-def check_niimg(niimg, atleast_4d=False):
+def _index_niimgs(niimgs, index):
```

Comment:
`niimg` is a Image object, so I guess the function should be called `_index_img` and the variable `img`.